hub/util/casting.py

```diff
@@ -37,6 +37,21 @@ def get_dtype(val: Union[np.ndarray, Sequence, Sample]) -> np.dtype:
         raise TypeError(f"Cannot infer numpy dtype for {val}")
 
 
+def get_htype(val: Union[np.ndarray, Sequence, Sample]) -> str:
```

Comment:
`samples` instead of `val` is easier to parse for me.

Reply:
Method is analogous to get_dtype.